### ISSUE TITLE
USER ISSUE: Unable to play due to invalid login

### ISSUE THREAD

**Issue by Eb-** [StrangeLoopGames/EcoIssues#24824]
\
**User Id: ()**
**Version:** 0.10.1.4 beta release-603

**Steps to Reproduce:**
Installed game after not playing since many versions back. Attempt to connect to a server. Get error: "Invalid login info.". it looks like my account was disabled after my first login attempt? Digging into the logs, it looks like it contacts the login server, authenticates, and then immediately after disconnects.

**Expected behavior:**
Let me log into the game and play

**Actual behavior:**
The following error message popped up:
```
Invalid login info.

<size=60.00%>Your account has been disabled.</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/14567094/Player.log)

**Comment by Eb-** [comment]
In this case, it is what it says on the tin, account is disabled. Found my old account (which i forgot about!) on the Eco site, and was unable to log into there.

This whole process could use improvement. A notification that an account has been disabled to the email of the account associated, a reason for the account being disabled, and steps towards re-enabling an account should be mailed out to the email address on file. In addition, in game there should be a more clear next steps on actions to perform to re-enable an account. It took longer than needed to find out I needed to email a support email address as a next step.
